src/topology/fiber_bundle.lean

```diff
@@ -240,6 +240,22 @@ begin
   rw [e.proj_symm_apply' h]
 end
 
+lemma symm_trans_symm (e e' : pretrivialization F proj) :
+  (e.to_local_equiv.symm.trans e'.to_local_equiv).symm =
+  e'.to_local_equiv.symm.trans e.to_local_equiv :=
```

Comment:
@Vierkantor, should this be part of the `equiv_like` API?

Reply:
Note that a local equiv is not `equiv_like`.

Reply:
Also, I haven't yet found a completely satisfactory way to express `symm` and `trans` in a totally generic way. So it's hard to say how to fit this in the `equiv_like` API.